tests/test_main.py

```diff
@@ -798,7 +798,6 @@ def test_param_n(self) -> None:
     def test_param_p(self) -> None:
         # internal parametrized tests for quick debug
         items = [  #
-            ("prod.py", b"secret_api_key='Ah\\tga%$FiQ@Ei8'", "secret_api_key", "Ah\\tga%$FiQ@Ei8"),  #
```

Comment:
It seems this PR makes some FPs...
Why those cases are removed?

Reply:
``\t``  sign did not appear in train set, so the sequence gives low ml probability. I'll update the sample instead removing